libs/sdk-embedding/src/iframe/ad.ts

```diff
@@ -176,6 +176,11 @@ export namespace EmbeddedAnalyticalDesigner {
          * Type represent that the remove filter context action is finished
          */
         RemoveFilterContextFinished = "removeFilterContextFinished",
+
+        /**
+         * Type notify KD that the editing insight has been canceled
+         */
+        VisualizationCancelled = "visualizationCancelled",
```

Comment:
The name is a bit confusing to me, it does not mention anything about editting. Should it be something like "insightEdittingCanceled"?

Reply:
The name `insightEdittingCanceled` look better than `visualizationCancelled`. What do you think @vu-pham ?

Reply:
I agree, and it should be insightEdittingCance**ll**ed

Reply:
correct name insightEdit**T**ingCanceled -> `insightEditingCanceled`